modules/GOTOPGO.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.preview.dsl=2
+
+process goTopGo {
+  input:
+    file DESeq2.tsv
```

Comment:
using `path` instead of `file` is now recommended.
You may want to change this into:

`path(DESeq2_in)`